src/state/slices/opportunitiesSlice/selectors.ts

```diff
@@ -139,9 +138,15 @@ export const selectUserStakingOpportunityByUserStakingId = createDeepEqualOutput
     const userOpportunity = userStakingOpportunities[userStakingId]
     const opportunityMetadata = stakingOpportunities[stakingId]
 
-    if (!(opportunityMetadata && userOpportunity)) return
+    if (!opportunityMetadata) return
 
     return {
+      // Overwritten by userOpportunity if it exists, else we keep defaulting to 0
```

Comment:
It's actually fine to have empty userData - we should still be able to select this and default to 0 amounts